CHANGELOG.md

```diff
@@ -13,7 +13,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * avifenc, avifdec: Allow "-j all" to automatically use all of the cores on the machine (#670)
 
 ### Changed
-
+* Re-map libavif speed 7-10 to libaom speed 7-9 (#682)
```

Comment:
We need to move this to line 8, under the "Unreleased" section.

Reply:
Done